src/components/Input/Input.tsx

```diff
@@ -92,7 +92,7 @@ export interface InputProps extends Omit<HTMLProps<HTMLInputElement>, 'label'> {
    */
   hideLabel?: boolean;
   /**
-   * Emotion style object to overwrite the <input> element styles.
+   * Emotion style object to overwrite the input element styles.
```

Comment:
This is to avoid this side-effect in Storybook docs:

<img width="998" alt="Screenshot 2020-11-27 at 15 42 29" src="https://user-images.githubusercontent.com/35560568/100460282-34f48f80-30c7-11eb-8995-bfecb84847fd.png">

Reply:
👀 That's funny and also surprising, I would expect them to sanitize (i.e. HTML-escape) the input.

Thanks for fixing it!